src/test/java/core/basesyntax/dao/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,148 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "UserLogin";
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final int DEFAULT_AGE = 20;
+    private static RegistrationService registrationService;
+    private User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User();
+        validUser.setPassword(DEFAULT_PASSWORD);
+        validUser.setAge(DEFAULT_AGE);
+        validUser.setLogin(DEFAULT_LOGIN);
+    }
+
+    @Test
+    void register_ValidUser_OK() {
+        User actualUser = registrationService.register(validUser);
+        assertEquals(validUser, actualUser);
+        assertNotNull(actualUser.getId());
+        assertTrue(Storage.people.contains(actualUser));
+    }
+
+    @Test
+    void age_UnderMinAge_NotOK() {
```

Comment:
```suggestion
    void register_UnderMinAge_NotOK() {
```
ми перевіряємо як працює метод register, тому першим словом в назві тесту краще використовувати register